api/server/server.go

```diff
@@ -20,12 +20,20 @@ import (
 
 type ToDDApi struct {
 	cfg config.Config
+	tdb db.DatabasePackage
```

Comment:
TODO - Ah, yes I see you placed this here. So, now I'm wondering how you're setting this?